src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -103,6 +103,8 @@ export default class ResultsViewPage extends React.Component<
 
         handleLegacySubmission(this.urlWrapper);
 
+        handlePostedSubmission(this.urlWrapper);
```

Comment:
So if I understand this correctly, only the ResultsViewPage can handle POST requests?

Reply:
that's correct.    the server writes the post data in all cases though so any page can in theory access it.